cloudsync/tests/fixtures/mock_provider.py

```diff
@@ -434,7 +477,8 @@ def _rename_single_object(self, source_object: MockFSObject, destination_path, *
         if self.oid_is_path:
             source_object.oid = destination_path
         self._store_object(source_object)
-        self._register_event(MockEvent.ACTION_RENAME, source_object, prior_oid)
+        if event:
```

Comment:
nit: Type annotation for `event`. Ambiguous if this a `bool` or an `Event`?

Reply:
yup, makes sense.